image-search/ViewControllers/ViewController.swift

```diff
@@ -2,33 +2,39 @@
 //  ViewController.swift
 //  image-search
 //
-//  Created by Renato on 9/25/19.
+//  Created by Renato Vieira on 9/25/19.
 //  Copyright © 2019 Stant. All rights reserved.
 //
 
 import UIKit
 
 
-func getNewAlbum() -> Album{
-    let albumGeneretor = AlbumGenerator()
-    let numberAlbum = Int.random(in: 0..<albumGeneretor.albumListing.count)
-    let newAlbum: Album = albumGeneretor.albumListing[numberAlbum]
-    
-    return newAlbum
-  
-    
-}
+
 
 
 class ViewController: UIViewController {
 
+    
+    let  activityIndicatorIndicator:UIActivityIndicatorView = UIActivityIndicatorView()
```

Comment:
Why is it named "activityIndicatorIndicator"? It should be activityIndicator only :)
Also separate the constant name of constant type. 
 
let  activityIndicator: UIActivityIndicatorView = UIActivityIndicatorView()

Reply:
I really don't know why I put twice indicator. Lack of attention :(